app/views/registrations/start.html.erb

```diff
@@ -15,6 +15,11 @@
     margin_bottom: 3,
   } %>
 
+  <%= sanitize(t("account_confusion.this_is_a_trial")) %>
+  <%= render "govuk_publishing_components/components/inset_text", {
+    text: sanitize(t("account_confusion.other_government_accounts"))
```

Comment:
Sorry, just another small thing I noticed: this doesn't need `sanitize`, does it?